raw_data/preprocessed/africa/eswatini_english.txt

```diff
@@ -0,0 +1,291 @@
+
+Cost Orders
+(67) [1] Where any person fails to comply with a requirement in an order, licence or approval issued under this Act, the Director may cause the required measures to be taken and may issue a cost order requiring that person to reimburse the Authority for the cost of taking the measures. [2] Any person to whom a cost order has been issued may, within 20 days of the order being served, apply for a review of the order in accordance with section 82. [3] If no application for the review of a cost order is made within the period mentioned in subsection [2], the cost order may be enforced as if it were an order of court.
+
+Offences
+(68) It is an offence for any person: [a] intentionally to obstruct an inspector in the exercise or performance of powers or duties under this Act; [b] intentionally or negligently to make a statement that is false or misleading in a material particular, where the statement is made: [i] in response to a requirement to furnish information under this Act; or [ii] for the purpose of obtaining the granting or variation of a licence, permit or consent under this Act; [c] intentionally to make a false entry in any record or document required to be kept or completed under this Act; [d] falsely to pretend to be an inspector; [e] to contravene this Act or to breach the conditions of a licence, approval or order issued or made under this Act.
+
+Liability of Managers of Bodies Corporate
+(69) [1] Where a body corporate commits an offence under this Act, every director or partner and any other person concerned in the management of the body corporate commits the offence unless that person proves that: [a] the offence was committed without his or her consent or connivance; and [b] the person exercised all such diligence to prevent the commission of the offence as ought to have been exercised by that person having regard to the nature of his or her functions in that capacity and to all the circumstances. [2] Every director or partner and any other person concerned in the management of, a body corporate to which a licence or order has been issued under this Act shall take all reasonable steps to prevent the body corporate from contravening or failing to comply with the licence or order. [3] A person who contravenes sub-section [2] commits an offence and shall, on conviction, be liable to imprisonment to a term not exceeding one year or to a fine not exceeding 50,000 Emalageni or to both. [4] Nothing in this section detracts from the liability or responsibility attached to any person under any other Act or law.
```

Comment:
Please replace `management of, a body corporate` with `management of a body corporate`. Also, please replace `Emalageni` with `Emalangeni`.